scripts/install_dependencies.py

```diff
@@ -63,6 +63,16 @@ def main():
 
     # Optionally, handle other dependencies via requirements.txt
     os.chdir(os.pardir)  # Move back to the root of the cloned repository directory structure
+
+    # Check if "python3-tk" is installed
+    result = subprocess.run(["dpkg", "-s", "python3-tk"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```

Comment:
This isn't going to work on windows. Can you provide an alternative for windows users?

Reply:
Added check for linux as pip can handle this for windows